src/coreclr/vm/jitinterface.cpp

```diff
@@ -7516,7 +7530,12 @@ static void getMethodInfoHelper(
             }
             else if (CoreLibBinder::IsClass(pMT, CLASS__RUNTIME_HELPERS))
             {
-                fILIntrinsic = getILIntrinsicImplementationForRuntimeHelpers(ftn, methInfo);
+                SigPointer localSig;
+                fILIntrinsic = getILIntrinsicImplementationForRuntimeHelpers(cxt, methInfo, &localSig);
+                if (fILIntrinsic)
+                {
+                    localSig.GetSignature(&pLocalSig, &cbLocalSig);
+                }
```

Comment:
```suggestion
                fILIntrinsic = getILIntrinsicImplementationForRuntimeHelpers(cxt, methInfo, &localSig);
```
and replace the remaining `localSig` locals and `pLocalSig`/`cbLocalSig` with a single `SigPointer localSig;` at the method scope.

Reply:
Done.